skpro/regression/bayesian/bayesian_conjugate.py

```diff
@@ -0,0 +1,231 @@
+"""Bayesian Conjugate Linear Regression Estimator."""
+
+__author__ = ["meraldoantonio"]
+
+import numpy as np
+import pandas as pd
+
+from skpro.distributions import Normal
+from skpro.regression.base import BaseProbaRegressor
+
+
+class BayesianConjugateLinearRegressor(BaseProbaRegressor):
+    """Bayesian probabilistic estimator for linear regression.
+
+    This estimator uses a multivariate Normal conjugate prior for the coefficients,
+    with the prior mean fixed at zero and precision controlled by a scalar alpha.
+    It assumes a known noise precision beta for the Gaussian likelihood.
+    For inference, it returns a multivariate Normal distribution as the posterior.
+    For prediction, it returns a series of univariate Normals for each data point.
+
```

Comment:
do you want to add math to make this precise? Also, the handling of the intercept should be discussed.